src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -36,17 +37,21 @@ public static final class DriveConstants {
     public static final int kGyroId = 15;
 
     // Chassis configuration
-    public static final double kTrackWidth = Units.inchesToMeters(26.5);
+    public static final Measure<Distance> kTrackWidth = Units.Inches.of(26.5);
 
     // Distance between centers of right and left wheels on robot
-    public static final double kWheelBase = Units.inchesToMeters(26.5);
+    public static final Measure<Distance> kWheelBase = Units.Inches.of(26.5);
+
+    public static final Measure<Distance> kWheelBaseRadius =
+        edu.wpi.first.units.Units.Inches.of(31.75 / 2);
+
     // Distance between front and back wheels on robot
     public static final SwerveDriveKinematics kDriveKinematics =
         new SwerveDriveKinematics(
-            new Translation2d(kWheelBase / 2, kTrackWidth / 2),
-            new Translation2d(kWheelBase / 2, -kTrackWidth / 2),
-            new Translation2d(-kWheelBase / 2, kTrackWidth / 2),
-            new Translation2d(-kWheelBase / 2, -kTrackWidth / 2));
+            new Translation2d(kWheelBase.in(Units.Meters) / 2, kTrackWidth.in(Units.Meters) / 2),
+            new Translation2d(kWheelBase.in(Units.Meters) / 2, -kTrackWidth.in(Units.Meters) / 2),
+            new Translation2d(-kWheelBase.in(Units.Meters) / 2, kTrackWidth.in(Units.Meters) / 2),
+            new Translation2d(-kWheelBase.in(Units.Meters) / 2, -kTrackWidth.in(Units.Meters) / 2));
```

Comment:
Why are you using meters here and inches elsewhere?

Reply:
think we usually measure things in inches, since that's what out tape measure is in, but most methods want meters